services/processor/native/get_code.go

```diff
@@ -90,7 +95,30 @@ func (s *service) retrieveDeployedContractInfoFromState(ctx context.Context, exe
 	return newContractInfo, nil
 }
 
-func (s *service) callGetCodeOfDeploymentSystemContract(ctx context.Context, executionContextId primitives.ExecutionContextId, contractName string) ([]byte, error) {
+func (s *service) getFullCodeOfDeploymentSystemContract(ctx context.Context, executionContextId primitives.ExecutionContextId, contractName string) ([]string, error) {
+	code, err := s.callGetCodeOfDeploymentSystemContract(ctx, executionContextId, contractName, 0)
```

Comment:
why do you treat the first part separately from the others?

Reply:
As I have explained to to Noam: backwards compatibility. I can refactor it if you want.